plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/nevergrad_sweeper.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from typing import List, Optional
+
+from hydra import TaskFunction
+from hydra.core.config_loader import ConfigLoader
+from hydra.plugins.sweeper import Sweeper
+from omegaconf import DictConfig
+
+from .config import OptimConf
+
+
+class NevergradSweeper(Sweeper):
+    """Class to interface with the Ax Platform"""
+
+    def __init__(
+        self, optim: OptimConf, version: int, parametrization: Optional[DictConfig]
+    ):
+        from .core import CoreNevergradSweeper
```

Comment:
Okay, I think once you rename to _core (or _impl) it would not be imported during plugin discovery.